src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -453,7 +451,7 @@ export const Details: React.FC<{
                             k8sVersion={appDetails.k8sVersion}
                         />
                     )}
-                    {isExternalToolAvailable && (
+                    {!appDetails?.deploymentAppDeleteRequest && isExternalToolAvailable && (
```

Comment:
Please perform check for !appDetails?.deploymentAppDeleteRequest once & wrap above elements under empty tag

Reply:
done